Dockerfile

```diff
@@ -1,51 +1,55 @@
 # Package path for this plugin module relative to the repo root
-ARG package=arcaflow_plugin_template_python
+ARG package=arcaflow_plugin_rtla
 
 # STAGE 1 -- Build module dependencies and run tests
 # The 'poetry' and 'coverage' modules are installed and verson-controlled in the
 # quay.io/arcalot/arcaflow-plugin-baseimage-python-buildbase image to limit drift
 FROM quay.io/arcalot/arcaflow-plugin-baseimage-python-buildbase:0.4.0 as build
 ARG package
 
+RUN dnf -y install rtla
+
 COPY poetry.lock /app/
 COPY pyproject.toml /app/
 
 # Convert the dependencies from poetry to a static requirements.txt file
-RUN python -m poetry install --without dev --no-root \
- && python -m poetry export -f requirements.txt --output requirements.txt --without-hashes
+RUN python3 -m poetry install --without dev --no-root \
+ && python3 -m poetry export -f requirements.txt --output requirements.txt --without-hashes
 
 COPY ${package}/ /app/${package}
 COPY tests /app/${package}/tests
 
 ENV PYTHONPATH /app/${package}
 WORKDIR /app/${package}
 
-# Run tests and return coverage analysis
-RUN python -m coverage run tests/test_${package}.py \
- && python -m coverage html -d /htmlcov --omit=/usr/local/*
+# # Run tests and return coverage analysis
+# RUN python -m coverage run tests/test_${package}.py \
+#  && python -m coverage html -d /htmlcov --omit=/usr/local/*
 
 
 # STAGE 2 -- Build final plugin image
 FROM quay.io/arcalot/arcaflow-plugin-baseimage-python-osbase:0.4.0
 ARG package
 
+RUN dnf -y install rtla
+
 COPY --from=build /app/requirements.txt /app/
-COPY --from=build /htmlcov /htmlcov/
+# COPY --from=build /htmlcov /htmlcov/
 COPY LICENSE /app/
 COPY README.md /app/
 COPY ${package}/ /app/${package}
 
 # Install all plugin dependencies from the generated requirements.txt file
-RUN python -m pip install -r requirements.txt
+RUN python -m pip install -r /app/requirements.txt
 
 WORKDIR /app/${package}
```

Comment:
I'm curious how the template works without this change?  Does it??  If so, why is this change necessary here?  If not, are you planning on fixing the template, too?  😇

Reply:
This was an artifact of some testing. The `/app/requirements.txt` path is how it is already used in the template.

Reply:
I don't understand:  the base version of `Dockerfile` is from the template, right?  And, you felt it necessary to modify the `pip install` source for this plugin.  Does that mean that the template is wrong?

Inspection suggests that the default working directory for `quay.io/arcalot/arcaflow-plugin-baseimage-python-osbase:0.4.2`, where I believe this `pip` command is being run, _is_ `/app`, so the command _should_ work just fine without your change.

It seems like a bad idea for boiler plate like this to differ from the template (unless there is a real problem, in which case maybe we should update the template (and, perhaps, the other copies of it...)).

Reply:
Sorry, I was mistaken in my last comment. Yes, it seems that `/app` is the working directory, so the build works fine regardless, but I have reverted this change.